instance/schemas/static_content_overrides.py

```diff
@@ -79,8 +82,17 @@ def fill_default_hero_text(text=None):
         )
         match = homepage_overlay_html_regex.match(text)
         if match:
-            title = match.group('title').strip() or DEFAULT_HERO_TITLE_TEXT
-            subtitle = match.group('subtitle').strip() or DEFAULT_HERO_SUBTITLE_TEXT
+            # Stores whatever value matched and overrides with default
+            # value when the frontend passes `reset_default`.
+            title = match.group('title')
+            if title == 'reset_default':
```

Comment:
That's a bit hacky. Let's hope that no user wouldn't want to have `reset_default` in the title nor subtitle :P
Maybe we could leave it as it was before? Or should we actually allow users to disable the title/subtitle?

Reply:
Yup, I'll revert this!

I'll make the reset button send back and empty text, that way the default value is filled.